### DISCUSSION TITLE
Why do I get that?

### DISCUSSION THREAD

**Discussion by ElianSimanca** [TheLastBen/fast-stable-diffusion#1102] [Q&A]
![image](https://user-images.githubusercontent.com/73455460/208888810-5ff0f284-0063-4895-8361-54452bb0765a.png)
![image](https://user-images.githubusercontent.com/73455460/208889023-0162d7a4-c3b1-41ff-bd68-224cbad0885c.png)

**Comment by TheLastBen** [comment]
huggingface added "feature_extractor" to the v2 models, if fixed it just now https://github.com/TheLastBen/fast-stable-diffusion/commit/6a099633764333befffb327f6985c15396570070
